frontend/views/components/GroupSettings/InviteLinks.vue

```diff
@@ -0,0 +1,250 @@
+<template lang="pug">
+page-section.c-section(:title='L("Invite links")')
+  template(#cta='')
+    button.button.is-small.is-outlined.c-active-button(
+      @click='switchList'
+    )
+      | {{ ephemeral.activeOn? L('Active links') : L('All links') }}
+      i.icon-angle-down.c-arrow
+
+  p.c-invite-description
+    i18n.has-text-1 Here's a list of all invite links you own
+
+  table.table.c-table(v-if='activeList.length')
+    thead
+      tr
+        i18n.name(tag='th') created for
+        i18n.invite-link(tag='th') invite link
+        i18n.state(tag='th') state
```

Comment:
@SebinSong now after our discussion you should have a better understanding of i18n, and see that i18n strings must be complete sentences, or the localizer will not have the context to translate them. So these 3 tags need to be collapsed into one. If you need HTML, use the `html=` attribute. Search the code for `html=` to see examples of how to pass HTML into i18n.